source/form.html

```diff
@@ -0,0 +1,226 @@
+                    <span class="checkbox__label">Черный</span>
+                  </input>
+                </label>
+              </li>
+              <li class="user-data__item">
+                <label class="checkbox">
+                  <input class="checkbox__input" name="color" type="checkbox" value="pink">
+                    <span class="checkbox__label">Розовый</span>
+                  </input>
+                </label>
+              </li>
+              <li class="user-data__item">
+                <label class="checkbox">
+                  <input class="checkbox__input" name="color" type="checkbox" value="orange">
+                    <span class="checkbox__label">Оранжевый</span>
+                  </input>
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset class="user-data__fieldset user-data__fieldset--user">
+            <legend class="user-data__legend">фио</legend>
+            <ul class="user-data__list">
+              <li class="user-data__item">
+                <label class="field">
+                  <span class="field__label">Имя:</span>
+                  <input class="field__input" type="text" name="last-name" placeholder="Введите ваше имя*">
+                    <span class="visually-hidden">Введите ваше имя*</span>
+                  </input>
+                </label>
+              </li>
+              <li class="user-data__item">
+                <label class="field">
+                  <span class="field__label">Фамилия:</span>
+                  <input class="field__input" type="text" name="second-name" placeholder="Укажите фамилию*">
+                    <span class="visually-hidden">Укажите фамилию*</span>
+                  </input>
+                </label>
+              </li>
+              <li class="user-data__item">
+                <label class="field">
+                  <span class="field__label">Отчество:</span>
+                  <input class="field__input" type="text" name="first-name" placeholder="Отчество, если желаете">
+                    <span class="visually-hidden">Отчество, если желаете</span>
+                  </input>
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset class="user-data__fieldset user-data__fieldset--contacts">
+            <legend class="user-data__legend">тел</legend>
+            <label class="contacts contacts--phone">
+              <input class="contacts__input" type="text" name="phone" placeholder="+7 ХХХ ХХХ ХХ ХХ*">
+                <span class="visually-hidden">Введите ваш номер телефона</span>
+              </input>
+            </label>
+            <legend class="user-data__legend">e-mail</legend>
+            <label class="contacts contacts--email">
+              <input class="contacts__input" type="text" name="email" placeholder="Ваш Email*">
```

Comment:
А тут создать тип для поля почты
```suggestion
              <input class="contacts__input" type="email" name="email" placeholder="Ваш Email*">
```